twikiget/cli.py

```diff
@@ -55,3 +72,102 @@ def wget_warc(url, wget_options, out_warc_file, directory_prefix):
+@click.option(
+    '--json',
+    'output_format',
+    flag_value='json',
+    default=None,
+    help='Get output in JSON format.')
+@click.option(
+    '--content-type',
+    'filter_content_type',
+    default='',
+    help='Filter files in an archive by content_type. '
+         'It can be either full version or just a begging of type name')
+@click.argument('warc_file')
+def list(warc_file, output_format, filter_content_type):
+    """ List files in WARC archive.
+
+    The list can be filtered by the HTTP Content Type,
+     and exported as json if needed.\n
+    Note that `content-type` option can be a full name of a type
+     or, to search broader, just the first part of it
+     e.g. `text/css` and `text`
+
+    Examples::
+
+        \t $ twikiget list ExampleTwiki.warc
+        \t $ twikiget list ExampleTwiki.warc --content-type=text/html
+        \t $ twikiget list ExampleTwiki.warc --content-type=text
+        \t $ twikiget list ExampleTwiki.warc --json
+    """
+    tablib_data = tablib.Dataset()
+    tablib_data.headers = ['uri', 'title', 'content_type']
+    with open(warc_file, 'rb') as fh:
+        for r in ArchiveIterator(fh):
+            if (r.rec_type == 'response' and
+                    r.http_headers.get_statuscode().startswith('2')):
+                title = None
+                content_type = r.http_headers.get_header('content-type')
+                uri = r.rec_headers.get_header('warc-target-uri')
+                # print(r.http_headers)
+                if content_type is None:
+                    content_type = 'application/octet-stream'
+                if content_type.lower().startswith(filter_content_type):
+                    if content_type.lower().startswith('text/html'):
+                        payload = r.content_stream().read()
+                        title = BeautifulSoup(
+                            payload, 'html.parser').find('title').getText()
+                    tablib_data.append(
+                        [str(uri), str(title), str(content_type)])
+
+        if output_format:
+            click.echo(tablib_data.export(output_format))
+        else:
+            table = PrettyTable()
+            table.field_names = ['uri', 'title', 'content_type']
+            for row in tablib_data:
+                table.add_row(row)
+            click.echo(table)
+
+
+@twikiget.command()
+@click.argument('warc-file')
+@click.argument('file-uri')
+def view(warc_file, file_uri):
+    """ View raw content of one of the files in WARC-FILE archive.
+
+    View command is usefull to inspect contents of one file from the archive.
+    It can be used with a pipe or a stream to view the file in a browser or
+    suitable program.
+    FILE-URI argument can be copied form the outputs of `twikiget list`.
+
+    Examples::
+
+        $ twikiget view ExampleTwiki.warc http://example.com/style.css \n
+          body {
+             margin: 0;
+             padding: 0;
+             font-family: Arial;
```

Comment:
Please remove the example output and keep just short example lines.  E.g. add one for "raw=on" because this one is useful.